features/login.feature

```diff
@@ -9,10 +9,10 @@ Feature: Log in as an admin, as a teacher, as a student
         As an admin, I want to be able to login
         When admin logs in with email <adminUsername> and password <password>
         Then admin accepts data protection
-        Then user-dashboard should have the correct initials
```

Comment:
why do you delete this line?

Reply:
i replaced it "Then the login must be successful", because this step does the same, we can delete one to avoid duplicate code